kubernetes-client/src/main/java/io/fabric8/kubernetes/client/informers/cache/Controller.java

```diff
@@ -157,7 +157,7 @@ private void processLoop() throws Exception {
       try {
         this.queue.pop(this.processFunc);
       } catch (InterruptedException t) {
-        log.warn("DefaultController#processLoop got interrupted {}", t.getMessage(), t);
+        log.debug("DefaultController#processLoop got interrupted: {}", t.getMessage());
```

Comment:
Just a side note: In some tests that I ran, the `t.getMessage()` just returned `null` which is not very informative.

Reply:
From experience I never saw something usefull as a message of an `InterruptedException`. I kept the message in the log to continue as much as possible the current behavior. And maybe some jvm implementations provide additionnal info ? I don't mind removing the message if required.

Reply:
Let's keep it, better that nothing.